### DISCUSSION TITLE
Multiple databases on multiple servers with the same schema.

### DISCUSSION THREAD

**Discussion by DwifteJB** [prisma/prisma#24978] [Q&A]
### Question

I want to have multiple servers in each region in the world (EU-WEST, EU-CENTRAL, US-CENTRAL, etc) and databases on those same servers which has data on, but I need to share data from thoes databases to the other databases, incase one goes down. Is there any way to do this easily in prisma, like having a central server that pushes data to different servers? Or any easier ways, if anyone can link me any articles or ways of doing this ill be appreciated. Thank you.

**Comment by ludralph** [comment]
Hi @DwifteJB 👋 

You can use database replication to keep your data synchronized across different regions. This involves setting up read replicas in different regions that can take over in case the primary database goes down. Prisma supports read replicas, which can help distribute read requests and improve resiliency. You can read more about this in the [Prisma blog on read replicas](https://www.prisma.io/blog/read-replicas-prisma-client-extension-f66prwk56wow#what-are-database-replicas-and-why-would-you-use-them).

**Comment by DwifteJB** [comment]
Thank you! I didn’t see this blog, i’ll definitely be implementing this!

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
